app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -19,12 +19,12 @@ class IrsAttemptsApiController < ApplicationController
 
     def create
       if timestamp
-        result = encrypted_security_event_log_result
+        send_data security_event_tokens
```

Comment:
For now, I am skipping over `encrypted_security_event_log_result` entirely, assuming we won't need to encrypt this file.

If that is true, we should delete the EnvelopeEncryptor file and the method here that calls it. If that's not true, we should restore this code.

Reply:
See suggestion on line 49 for more context for this. 

```suggestion
        send_data security_event_tokens.values.join("/r/n")
```